unlock-protocol-com/blog/subgraph-v2.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Subgraph upgrade
+subTitle: An upgraded version of the subgraph is now available
```

Comment:
please add another metadata line for description
**description: An upgraded version of the subgraph is now available for Unlock Protocol using The Graph.**